.travis.yml

```diff
@@ -33,25 +33,32 @@ matrix:
     - php: 7.3
       env:
         - COMPOSER_FLAGS="--prefer-stable --prefer-lowest" DEPENDENCIES="symfony/validator:^3.4.4 symfony/templating:^3.4.4"
+    - php: 7.4
```

Comment:
this should be php 8. we want to do the lowest versions build with the lowest and the most recent php version we support.